src/HealthGPS/static_linear_model.cpp

```diff
@@ -513,10 +564,13 @@ StaticLinearModelDefinition::StaticLinearModelDefinition(
 
 std::unique_ptr<RiskFactorModel> StaticLinearModelDefinition::create_model() const {
     return std::make_unique<StaticLinearModel>(
-        expected_, expected_trend_, trend_steps_, expected_trend_boxcox_, names_, models_, ranges_,
-        lambda_, stddev_, cholesky_, policy_models_, policy_ranges_, policy_cholesky_,
-        trend_models_, trend_ranges_, trend_lambda_, info_speed_, rural_prevalence_, income_models_,
-        physical_activity_stddev_);
+        std::make_shared<RiskFactorSexAgeTable>(*expected_),
+        std::make_shared<std::unordered_map<core::Identifier, double>>(*expected_trend_),
```

Comment:
I would make these pointers in `model_parser`, like how we've done with the  `trend_`... variables in `model_parser.cpp`, lines 236-238. Less chance for memory corruptions in future.